File: MissingNoArray.c
#include <stdio.h>

int main() {
    int n;

    printf("Enter n (range 1 to n): ");
    scanf("%d", &n);

    int arr[n - 1];
    int actualSum = 0;

    printf("Enter %d numbers:\n", n - 1);
    for (int i = 0; i < n - 1; i++) {
        scanf("%d", &arr[i]);
        actualSum += arr[i];
    }

    int expectedSum = n * (n + 1) / 2;
    int missing = expectedSum - actualSum;

    printf("Missing number: %d", missing);

    return 0;
}

File: MissingNoArray.py
n = int(input("Enter n (range 1 to n): "))

print("Enter", n-1, "numbers:")
arr = []

for _ in range(n - 1):
    arr.append(int(input()))

expected_sum = n * (n + 1) // 2
actual_sum = sum(arr)

missing = expected_sum - actual_sum

print("Missing number:", missing)
